machine-learning/solutions/energy_price_forecasting/data/README.md

```diff
@@ -0,0 +1,28 @@
+# Disclaimer
+
+The data used to train this solution has been downloaded from http://complatt.smartwatt.net/
+This website hosts this data for a closed competition meant to solve the energegy price forecasting problem.
```

Comment:
spelling

Reply:
you should also make sure that data use is approved, if Google doesn't own this data